### ISSUE TITLE
Package fails import on Python 3.12, missing dependencies

### ISSUE THREAD

**Issue by remyjette** [microsoft/azure-devops-python-api#470]
Package should declare a dependency on pkg_resources if it is using it

Installed `azure_devops-7.1.0b3-py3-none-any.whl` on a fresh machine (using `pip install azure-devops`)

```
>>> import azure.devops
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\Remy\AppData\Roaming\Python\Python312\site-packages\azure\__init__.py", line 5, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

**Comment by remyjette** [comment]
Fixed in #469

**Comment by nechvatalp** [comment]
The fix has been merged closing the issue, thanks for reporting.
